Fedora/input/fixes/bash/disable_prelink.sh

```diff
@@ -5,8 +7,8 @@ if grep -q ^PRELINKING /etc/sysconfig/prelink
 then
   sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
 else
-  echo -e "\n# Set PRELINKING=no per security requirements" >> /etc/sysconfig/prelink
-  echo "PRELINKING=no" >> /etc/sysconfig/prelink
+  append_to_file "\n# Set PRELINKING=no per security requirements" "/etc/sysconfig/prelink"
+  append_to_file "PRELINKING=no" "/etc/sysconfig/prelink"
```

Comment:
I like the idea of an `````append_to_file`````, but this script is an example of where it breaks. A line break is undesired between the two lines here.

Perhaps it's time to start a coding standards page, calling for things like "when remediation calls for inserting newlines into files, a comment should be made indicating the CCE number prior to the newline. for example:

# Per CCE-12345: Set prelinking=no in /etc/sysconfig/prelink
PRELINKING=no 
`````